src/IssueViz.Security.UnitTests/Hotspots/LocalHotspotStoreTests.cs

```diff
@@ -336,6 +336,35 @@ public void Refresh_NewServerHotspots_ExistingHotspotsRematched()
             .BeEquivalentTo(new IssuesChangedEventArgs(issueVisualizations, issueVisualizations));
     }
 
+    [TestMethod]
```

Comment:
Would this be simpler as a data-driven test with parameters `status`, `resolution`, `shouldBeIncluded` ?